README.md

```diff
@@ -745,6 +745,33 @@ Current phase of backup restoration. Possible options are these:
   automatically but it may fail repeatedly and analyzing logs to get more
   details regarding the failure is advisable.
 
+Running tests
+=============
```

Comment:
Can you add a section about running the tests without docker too ?

Reply:
Makes sense, will do.